os/arch/arm/src/armv7-r/arm_syscall.c

```diff
@@ -526,6 +526,7 @@ uint32_t *arm_syscall(uint32_t *regs)
 	lldbg("SYSCALL from 0x%x\n", regs[REG_PC]);
 	current_regs = regs;
 	PANIC();
+	return regs;
```

Comment:
Execution will never reach this point, since we are calling PANIC just before. 
Since we added this line to fix compile error, maybe we could add a comment explaining the purpose.

Reply:
as PANIC() returns now, reverted these change

Reply:
since PANIC() returns, we don't need this change, reverted it

Reply:
I can see adding `return regs;` yet. Will you remove it or not?
```
Since we added this line to fix compile error, maybe we could add a comment explaining the purpose.
```
Isn't it necessary?